src/main/java/org/tron/common/entity/NodeInfo.java

```diff
@@ -22,6 +24,8 @@
   /*machine information*/
   private MachineInfo machineInfo;
 
+  private Map<String, String> cheatWitnessInfoMap = new HashMap<>();
+
   public static class MachineInfo {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a constructor to the class, or provide default values. ([squid:S1258](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1258))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-vKlHK86Hco_1fN&open=AWalU-vKlHK86Hco_1fN)

[ISSUE_KEY]: AWalU-vKlHK86Hco_1fN